### DISCUSSION TITLE
osu bug with tablet

### DISCUSSION THREAD

**Discussion by Ithoo235** [ppy/osu#24413] [Q&A]
the bug allows you to do spinner in a speed of 3000 rpm
to do the bug u need a tablet and use the osu tablet config. on the tablet itself make sure the whole tablet is usable while in osu small. when u put the pen down it jumps back and forth at extreme speeds.
i usually hover the pen and this only happens when the pen is touching the tablet

https://cdn.discordapp.com/attachments/817404219492794448/1134854354768236657/insane_osu_bug.mp4
(sorry ab needing download)

the map i played on to show this only has 3 spinners so you wont get as many points as i did on a normal map.

Lazer 2023.717.0

**Comment by n0Oo0Oo0b** [comment]
https://github.com/ppy/osu/issues/8224#issuecomment-674810230

**Reply by Ithoo235** [reply]
still tho im able to cheese them beyond fast

**Reply by n0Oo0Oo0b** [reply]
My point was that this discussion is a duplicate of that issue, which just adds noise for the developers. If you think this adds more detail that could be helpful to that issue, then you could post a comment there instead.

**Reply by Ithoo235** [reply]
i read it and there is nthn i can do about it anyways
